style.css

```diff
@@ -3,4 +3,486 @@ body {
+  padding-left: 0.7em;
+  width: 20vw;
+  height: 10vh;
+}
+
+.fa-bars {
+  font-size: 1.3rem;
+}
+
+.logo > img {
+  flex-shrink: 1;
+  width: 7vw;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1em;
+  width: 50%;
+  top: 10px;
+}
+
+.mic {
+  font-size: 1.2rem;
+}
+
+.search > .search-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  height: 6vh;
+  border: 2px solid #ddd;
+  border-radius: 20px;
+}
+
+.search > .search-content:focus {
+  border: none;
+}
+
+.search > .search-content:active {
+  border: none;
+}
+
+.search-content > .fa-search {
+  font-size: 1.5rem;
+  padding-left: 1em;
+}
+
+.search-content > input {
+  margin-left: 1em;
+  width: 90%;
+  height: 5vh;
+  border: none;
+}
+
+.navbuttons {
+  display: flex;
+  box-sizing: border-box;
+  padding-right: 2em;
+  padding-top: 0.5em;
+  width: 20%;
+  height: 8vh;
+  justify-content: flex-end;
+  gap: 0.4em;
+  align-items: center;
+}
+
+.buttons-header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 2.5em;
+  height: 2.5em;
+  border-radius: 50%;
+}
+
+.buttons-header > img {
+  flex-shrink: 0;
+}
+
+.material-symbols-outlined {
+  font-size: 2em;
+}
+
+.search-content > div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 10%;
+  height: 100%;
+  background-color: rgb(248, 248, 248);
+  border-left: 2px solid #ddd;
+  border-radius: 0 40px 40px 0;
+}
+
+.buttons-header:hover {
+  background-color: rgb(240, 240, 240);
+}
+
+.search-content > div:hover {
+  background-color: rgb(240, 240, 240);
+}
+
+.mic:hover {
+  background-color: rgb(240, 240, 240);
+}
+
+.sidebar-item:hover {
+  background-color: rgb(240, 240, 240);
+  border-radius: 10px;
+}
+
+.sidebar::-webkit-scrollbar {
+  display: none;
+  width: 8px;
+}
+
+.sidebar::-webkit-scrollbar-thumb {
+  background: #909090;
+  border-radius: 4px;
+}
+
+.sidebar::-webkit-scrollbar-track {
+  background: #fff;
+  border-radius: 4px;
+}
+
+.sidebar::-webkit-scrollbar-thumb:hover {
+  background: #909090;
+  box-shadow: 0 0 2px 1px rgba(0, 0, 0, 0.2);
+}
+
+.sidebar::-webkit-scrollbar-track:hover,
+.sidebar::-webkit-scrollbar-track:active {
+  background: #fff;
+}
+
+.sidebar:hover::-webkit-scrollbar {
+  display: initial;
+}
+
+.sidebar::-webkit-scrollbar-thumb:active {
+  background-color: #909090;
+}
+
+html::-webkit-scrollbar {
+  width: 8px;
+}
+
+html::-webkit-scrollbar-track {
+  background: #fff;
+  border-radius: 4px;
+}
+
+html::-webkit-scrollbar-thumb {
+  background: #606060;
+  border-radius: 4px;
+}
+
+html::-webkit-scrollbar-thumb:hover {
+  background: #606060;
+  box-shadow: 0 0 2px 1px rgba(0, 0, 0, 0.2);
+}
+
+html::-webkit-scrollbar-track:hover,
+html::-webkit-scrollbar-track:active {
+  background: #fff;
+}
+
+html:hover::-webkit-scrollbar {
+  display: initial;
+}
+
+html::-webkit-scrollbar-thumb:active {
+  background-color: #909090;
+}
```

Comment:
Only a suggestion guys, this part could be placed on the top of this file, is about `html`, so is nice see the natives elements at the first then our customs classes